chrome/content/zotero/zoteroPane.js

```diff
@@ -3110,7 +3110,7 @@ var ZoteroPane = new function()
 			document.getElementById(ZoteroPane.itemsView.id).focus();
 		}
 		
-		Zotero_Tabs.select('zotero-pane');
+		Zotero_Tabs.select('zotero-pane', false, { zpRefocusID: ZoteroPane.itemsView.id });
```

Comment:
`zpRefocusID` seems overly concerned with the internals of `Zotero_Tabs.select()`. Shouldn't it just be, say, `focusElementID`?